trade-finance-manager-api/src/v1/helpers/create-tasks-amendment.helper-getTasksAssignedToUserByGroup.test.js

```diff
@@ -0,0 +1,53 @@
+const { getTasksAssignedToUserByGroup } = require('./create-tasks-amendment.helper');
+const { TEAMS, TASKS } = require('../../constants');
+
+const api = require('../api');
+const { mockFindUserById } = require('../__mocks__/common-api-mocks');
+const {
+  MOCK_USERS_FOR_TASKS,
+  MOCK_TASKS,
+  TASKS_ASSIGNED_TO_UNDERWRITER,
+  TASKS_ASSIGNED_TO_UNDERWRITER_MANAGER,
+  TASKS_ASSIGNED_TO_UNDERWRITER_AND_UNDERWRITER_MANAGER,
+  TASKS_UNASSIGNED_FOR_UNDERWRITER,
+  MOCK_TASKS_CHANGE_TIME,
+} = require('../__mocks__/mock-amendment-tasks-assign-by-team');
+
+jest.mock('../api');
+
+describe('getTasksAssignedToUserByGroup()', () => {
+  const { underwriter, underwriterManager } = MOCK_USERS_FOR_TASKS;
+
+  beforeEach(() => {
+    api.findUserById.mockReset();
+    mockFindUserById();
+
+    jest.useFakeTimers();
+    jest.setSystemTime(MOCK_TASKS_CHANGE_TIME * 1000); // Tuesday, December 23, 2008 2:40:00 AM GMT
+  });
+
+  it('should assign tasks to Underwriter', async () => {
```

Comment:
Can we add an assertion with empty arguments please to check the error handling please.

Reply:
I added validation to helper and unit tests